sharpy/linear/src/lingebm.py

```diff
@@ -14,6 +14,7 @@
 import sharpy.utils.cout_utils as cout
 import sharpy.structure.utils.modalutils as modalutils
 import warnings
+from sharpy.linear.utils.ss_interface import LinearVector, StateVariable, OutputVariable, InputVariable
 
 
 class FlexDynamic():
```

Comment:
How well the linearisation plays with the polars corrections in the aero? It should be all right since the linearised external forcing terms are taken from the .structure.steady_app_forces which includes those...

Reply:
A few things are missing, yes, the steady forces with the corrections are included but these only appear in the stiffening and damping terms added to the UVLM (i.e. geometric effects only). The UVLM matrices would be missing the updated-with-polar circulation at the ref state.

The other correction is done on the output of the aeroelastic system, with the polars being modified in stability axes to change the lift only at this stage. I just realised I have this elsewhere so I'll merge it in a separate PR in a second.

This is Sec 4.1.1 and Ch8 further work of my thesis :)